Kudu.Core/Deployment/Generator/ExternalCommandBuilder.cs

```diff
@@ -129,7 +128,7 @@ public IList<string> GetPostBuildActionScripts()
             var defaultPath = Path.Combine(Environment.DeploymentToolsPath, PostDeploymentActions);
             if (!string.IsNullOrEmpty(System.Environment.GetEnvironmentVariable(SettingsKeys.PostDeploymentActionsDirectory)))
```

Comment:
yep, will do. just confirmed from our old code (reset and deploy), we do support settings from .deployment file, but we only expect absolute file path.

i will update the logic to honor settings from both environment variable and deployment setting, relative path and absolute path.